lib/logidze/versioned_association.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+require 'active_support'
+
+module Logidze
+  module VersionedAssociation
+    module SingularAssociation
+      def reader(force_reload = false)
```

Comment:
It turned out, in case of singular association we can use `reader` method to version instead of `load_target`. It will save us from the problem of already loaded associations, and multiple invocation won't make us suffer, because logidze won't do anything on the similar `at` calls